### ISSUE TITLE
[Bug] YoloX-S onnx converted in Docker has vector size error

### ISSUE THREAD

**Issue by jamestkpoon** [open-mmlab/mmdeploy#2191]
### Checklist

- [X] I have searched related issues but cannot get the expected help.
- [X] 2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/tree/main/docs/en/faq.md) but cannot get the expected help.
- [X] 3. The bug has not been fixed in the latest version.

### Describe the bug

I am using the latest ubuntu20.04-cuda11.3-mmdeploy Docker image, to try and get an .onnx from a YoloX-S .pth trained in an mmdetection container. The only modifications to the YoloX config are image size ([300,300]) and number of classes (1).
Contents of config files are available below, under "Reproduction". The .onnx appears to be generated, but I then get an error (log in "Error traceback").

Some additional notes:
In mmdeploy, when I first run  ```python3 tools/deploy.py ...``` I encounter the error ```ImportError: mmdet has not been installed. Import mmdet failed.```. ```mim install mmdet``` seems to fix this, but raises some warnings later:
```
06/15 08:52:55 - mmengine - INFO - Start pipeline mmdeploy.apis.pytorch2onnx.torch2onnx in subprocess
06/15 08:52:56 - mmengine - WARNING - Failed to search registry with scope "mmdet" in the "Codebases" registry tree. As a workaround, the current "Codebases" registry in "mmdeploy" is used to build instance. This may cause unexpected failure when running the built modules. Please check whether "mmdet" is a correct scope, or whether the registry is initialized.
06/15 08:52:56 - mmengine - WARNING - Failed to search registry with scope "mmdet" in the "mmdet_tasks" registry tree. As a workaround, the current "mmdet_tasks" registry in "mmdeploy" is used to build instance. This may cause unexpected failure when running the built modules. Please check whether "mmdet" is a correct scope, or whether the registry is initialized.
Loads checkpoint by local backend from path: /checkpoint_dir/epoch_300.pth
06/15 08:52:56 - mmengine - WARNING - DeprecationWarning: get_onnx_config will be deprecated in the future. 
06/15 08:52:56 - mmengine - INFO - Export PyTorch model to ONNX: /root/workspace/mmdeploy/detector.onnx.
06/15 08:52:56 - mmengine - WARNING - Can not find torch._C._jit_pass_onnx_autograd_function_process, function rewrite will not be applied
```

### YoloX-S training config
```
"""
Adapted from /mmdetection/configs/yolox/yolox_s_8xb8-300e_coco.py
"""

_base_ = [
    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py',
    './yolox_tta.py'
]

img_scale = (300, 300)  # width, height

# model settings
model = dict(
    type='YOLOX',
    data_preprocessor=dict(
        type='DetDataPreprocessor',
        pad_size_divisor=32,
        batch_augments=[
            dict(
                type='BatchSyncRandomResize',
                random_size_range=(100, 500),
                size_divisor=32,
                interval=10)
        ]),
    backbone=dict(
        type='CSPDarknet',
        deepen_factor=0.33,
        widen_factor=0.5,
        out_indices=(2, 3, 4),
        use_depthwise=False,
        spp_kernal_sizes=(5, 9, 13),
        norm_cfg=dict(type='BN', momentum=0.03, eps=0.001),
        act_cfg=dict(type='Swish'),
    ),
    neck=dict(
        type='YOLOXPAFPN',
        in_channels=[128, 256, 512],
        out_channels=128,
        num_csp_blocks=1,
        use_depthwise=False,
        upsample_cfg=dict(scale_factor=2, mode='nearest'),
        norm_cfg=dict(type='BN', momentum=0.03, eps=0.001),
        act_cfg=dict(type='Swish')),
    bbox_head=dict(
        type='YOLOXHead',
        num_classes=1,
        in_channels=128,
        feat_channels=128,
        stacked_convs=2,
        strides=(8, 16, 32),
        use_depthwise=False,
        norm_cfg=dict(type='BN', momentum=0.03, eps=0.001),
        act_cfg=dict(type='Swish'),
        loss_cls=dict(
            type='CrossEntropyLoss',
            use_sigmoid=True,
            reduction='sum',
            loss_weight=1.0),
        loss_bbox=dict(
            type='IoULoss',
            mode='square',
            eps=1e-16,
            reduction='sum',
            loss_weight=5.0),
        loss_obj=dict(
            type='CrossEntropyLoss',
            use_sigmoid=True,
            reduction='sum',
            loss_weight=1.0),
        loss_l1=dict(type='L1Loss', reduction='sum', loss_weight=1.0)),
    train_cfg=dict(assigner=dict(type='SimOTAAssigner', center_radius=2.5)),
    # In order to align the source code, the threshold of the val phase is
    # 0.01, and the threshold of the test phase is 0.001.
    test_cfg=dict(score_thr=0.01, nms=dict(type='nms', iou_threshold=0.65)))

# dataset settings
data_root = '/data/'
dataset_type = 'CocoDataset'
metainfo = {"classes": ["pedestrian_crossing_light"]}

# Example to use different file client
# Method 1: simply set the data root and let the file I/O module
# automatically infer from prefix (not support LMDB and Memcache yet)

# data_root = 's3://openmmlab/datasets/detection/coco/'

# Method 2: Use `backend_args`, `file_client_args` in versions before 3.0.0rc6
# backend_args = dict(
#     backend='petrel',
#     path_mapping=dict({
#         './data/': 's3://openmmlab/datasets/detection/',
#         'data/': 's3://openmmlab/datasets/detection/'
#     }))
backend_args = None

train_pipeline = [
    dict(type='Mosaic', img_scale=img_scale, pad_val=114.0),
    dict(
        type='RandomAffine',
        scaling_ratio_range=(0.1, 2),
        # img_scale is (width, height)
        border=(-img_scale[0] // 2, -img_scale[1] // 2)),
    dict(
        type='MixUp',
        img_scale=img_scale,
        ratio_range=(0.8, 1.6),
        pad_val=114.0),
    dict(type='YOLOXHSVRandomAug'),
    dict(type='RandomFlip', prob=0.5),
    # According to the official implementation, multi-scale
    # training is not considered here but in the
    # 'mmdet/models/detectors/yolox.py'.
    # Resize and Pad are for the last 15 epochs when Mosaic,
    # RandomAffine, and MixUp are closed by YOLOXModeSwitchHook.
    dict(type='Resize', scale=img_scale, keep_ratio=True),
    dict(
        type='Pad',
        pad_to_square=True,
        # If the image is three-channel, the pad value needs
        # to be set separately for each channel.
        pad_val=dict(img=(114.0, 114.0, 114.0))),
    dict(type='FilterAnnotations', min_gt_bbox_wh=(1, 1), keep_empty=False),
    dict(type='PackDetInputs')
]

train_dataset = dict(
    # use MultiImageMixDataset wrapper to support mosaic and mixup
    type='MultiImageMixDataset',
    dataset=dict(
        type=dataset_type,
        data_root=data_root,
        ann_file='training/labels.json',
        data_prefix=dict(img='training/data/'),
        metainfo=metainfo,
        pipeline=[
            dict(type='LoadImageFromFile', backend_args=backend_args),
            dict(type='LoadAnnotations', with_bbox=True)
        ],
        filter_cfg=dict(filter_empty_gt=False, min_size=32),
        backend_args=backend_args),
    pipeline=train_pipeline)

test_pipeline = [
    dict(type='LoadImageFromFile', backend_args=backend_args),
    dict(type='Resize', scale=img_scale, keep_ratio=True),
    dict(
        type='Pad',
        pad_to_square=True,
        pad_val=dict(img=(114.0, 114.0, 114.0))),
    dict(type='LoadAnnotations', with_bbox=True),
    dict(
        type='PackDetInputs',
        meta_keys=('img_id', 'img_path', 'ori_shape', 'img_shape',
                   'scale_factor'))
]

train_dataloader = dict(
    batch_size=8,
    num_workers=4,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=True),
    dataset=train_dataset)
val_dataloader = dict(
    batch_size=8,
    num_workers=4,
    persistent_workers=True,
    drop_last=False,
    sampler=dict(type='DefaultSampler', shuffle=False),
    dataset=dict(
        type=dataset_type,
        data_root=data_root,
        ann_file='validation/labels.json',
        data_prefix=dict(img='validation/data/'),
        metainfo=metainfo,
        test_mode=True,
        pipeline=test_pipeline,
        backend_args=backend_args))
test_dataloader = val_dataloader

val_evaluator = dict(
    type='CocoMetric',
    ann_file=data_root + 'validation/labels.json',
    metric='bbox',
    backend_args=backend_args)
test_evaluator = val_evaluator

# training settings
max_epochs = 300
num_last_epochs = 15
interval = 10

train_cfg = dict(max_epochs=max_epochs, val_interval=interval)

# optimizer
# default 8 gpu
base_lr = 0.005
optim_wrapper = dict(
    type='OptimWrapper',
    optimizer=dict(
        type='SGD', lr=base_lr, momentum=0.9, weight_decay=5e-4,
        nesterov=True),
    paramwise_cfg=dict(norm_decay_mult=0., bias_decay_mult=0.))

# learning rate
param_scheduler = [
    dict(
        # use quadratic formula to warm up 5 epochs
        # and lr is updated by iteration
        # TODO: fix default scope in get function
        type='mmdet.QuadraticWarmupLR',
        by_epoch=True,
        begin=0,
        end=5,
        convert_to_iter_based=True),
    dict(
        # use cosine lr from 5 to 285 epoch
        type='CosineAnnealingLR',
        eta_min=base_lr * 0.05,
        begin=5,
        T_max=max_epochs - num_last_epochs,
        end=max_epochs - num_last_epochs,
        by_epoch=True,
        convert_to_iter_based=True),
    dict(
        # use fixed lr during last 15 epochs
        type='ConstantLR',
        by_epoch=True,
        factor=1,
        begin=max_epochs - num_last_epochs,
        end=max_epochs,
    )
]

default_hooks = dict(
    checkpoint=dict(
        interval=interval,
        max_keep_ckpts=3  # only keep latest 3 checkpoints
    ))

custom_hooks = [
    dict(
        type='YOLOXModeSwitchHook',
        num_last_epochs=num_last_epochs,
        priority=48),
    dict(type='SyncNormHook', priority=48),
    dict(
        type='EMAHook',
        ema_type='ExpMomentumEMA',
        momentum=0.0001,
        update_buffers=True,
        priority=49)
]

# NOTE: `auto_scale_lr` is for automatically scaling LR,
# USER SHOULD NOT CHANGE ITS VALUES.
# base_batch_size = (8 GPUs) x (8 samples per GPU)
auto_scale_lr = dict(base_batch_size=64)

load_from = "/data/yolox_s_8x8_300e_coco_20211121_095711-4592a793.pth"
```

### mmdeploy config
```
codebase_config = dict(
    type="mmdet",
    task="ObjectDetection",
    post_processing=dict(
        max_output_boxes_per_class=2, iou_threshold=0.65, score_threshold=0.3, pre_top_k=10, keep_top_k=True
    ),
)

backend_config = dict(
    type="onnxruntime",
    model_inputs=[
        dict(
            input_shapes=dict(
                input=dict(min_shape=[1, 3, 300, 300], opt_shape=[4, 3, 300, 300], max_shape=[64, 3, 300, 300])
            )
        )
    ],
)

onnx_config = dict(
    type="onnx",
    dynamic_axes={"input": {0: "batch", 2: "height", 3: "width"}, "output": {0: "batch"}},
    export_params=True,
    keep_initializers_as_inputs=False,
    opset_version=11,
    save_file="detector.onnx",
    input_names=["input"],
    output_names=["output1", "output2"],
    input_shape=[300, 300],
    optimize=True,
)
```

### Environment

```Shell
06/15 10:37:12 - mmengine - INFO - **********Environmental information**********
06/15 10:37:13 - mmengine - INFO - sys.platform: linux
06/15 10:37:13 - mmengine - INFO - Python: 3.8.10 (default, Mar 13 2023, 10:26:41) [GCC 9.4.0]
06/15 10:37:13 - mmengine - INFO - CUDA available: True
06/15 10:37:13 - mmengine - INFO - numpy_random_seed: 2147483648
06/15 10:37:13 - mmengine - INFO - GPU 0: NVIDIA GeForce RTX 4050 Laptop GPU
06/15 10:37:13 - mmengine - INFO - CUDA_HOME: /usr/local/cuda
06/15 10:37:13 - mmengine - INFO - NVCC: Cuda compilation tools, release 11.3, V11.3.109
06/15 10:37:13 - mmengine - INFO - GCC: x86_64-linux-gnu-gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
06/15 10:37:13 - mmengine - INFO - PyTorch: 1.10.0+cu113
06/15 10:37:13 - mmengine - INFO - PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.2.3 (Git Hash 7336ca9f055cf1bfa13efb658fe15dc9b41f0740)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.3
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.2
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.3, CUDNN_VERSION=8.2.0, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.10.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 

06/15 10:37:13 - mmengine - INFO - TorchVision: 0.11.0+cu113
06/15 10:37:13 - mmengine - INFO - OpenCV: 4.5.4
06/15 10:37:13 - mmengine - INFO - MMEngine: 0.7.4
06/15 10:37:13 - mmengine - INFO - MMCV: 2.0.0
06/15 10:37:13 - mmengine - INFO - MMCV Compiler: GCC 9.3
06/15 10:37:13 - mmengine - INFO - MMCV CUDA Compiler: 11.3
06/15 10:37:13 - mmengine - INFO - MMDeploy: 1.1.0+264de4d
06/15 10:37:13 - mmengine - INFO - 

06/15 10:37:13 - mmengine - INFO - **********Backend information**********
06/15 10:37:13 - mmengine - INFO - tensorrt:	8.2.3.0
06/15 10:37:13 - mmengine - INFO - tensorrt custom ops:	Available
06/15 10:37:13 - mmengine - INFO - ONNXRuntime:	None
06/15 10:37:13 - mmengine - INFO - ONNXRuntime-gpu:	1.8.1
06/15 10:37:13 - mmengine - INFO - ONNXRuntime custom ops:	Available
06/15 10:37:13 - mmengine - INFO - pplnn:	0.8.1
06/15 10:37:13 - mmengine - INFO - ncnn:	1.0.20230224
06/15 10:37:13 - mmengine - INFO - ncnn custom ops:	Available
06/15 10:37:13 - mmengine - INFO - snpe:	None
06/15 10:37:13 - mmengine - INFO - openvino:	2022.3.0
06/15 10:37:13 - mmengine - INFO - torchscript:	1.10.0+cu113
06/15 10:37:13 - mmengine - INFO - torchscript custom ops:	Available
06/15 10:37:13 - mmengine - INFO - rknn-toolkit:	None
06/15 10:37:13 - mmengine - INFO - rknn-toolkit2:	None
06/15 10:37:13 - mmengine - INFO - ascend:	None
06/15 10:37:13 - mmengine - INFO - coreml:	None
06/15 10:37:13 - mmengine - INFO - tvm:	None
06/15 10:37:13 - mmengine - INFO - vacc:	None
06/15 10:37:13 - mmengine - INFO - 

06/15 10:37:13 - mmengine - INFO - **********Codebase information**********
06/15 10:37:13 - mmengine - INFO - mmdet:	3.0.0
06/15 10:37:13 - mmengine - INFO - mmseg:	None
06/15 10:37:13 - mmengine - INFO - mmpretrain:	None
06/15 10:37:13 - mmengine - INFO - mmocr:	None
06/15 10:37:13 - mmengine - INFO - mmagic:	None
06/15 10:37:13 - mmengine - INFO - mmdet3d:	None
06/15 10:37:13 - mmengine - INFO - mmpose:	None
06/15 10:37:13 - mmengine - INFO - mmrotate:	None
06/15 10:37:13 - mmengine - INFO - mmaction:	None
06/15 10:37:13 - mmengine - INFO - mmrazor:	None
```

## Error traceback

```Shell
06/15 08:53:03 - mmengine - INFO - Successfully loaded onnxruntime custom ops from /root/workspace/mmdeploy/mmdeploy/lib/libmmdeploy_onnxruntime_ops.so
2023-06-15 08:53:03.673777200 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1434'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673791705 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1433'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673794932 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1431'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673797597 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1428'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673799865 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1422'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673802316 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1418'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673804571 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1398'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673806983 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1365'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673809297 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1351'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673811818 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1410'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673815945 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1406'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673837483 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1387'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673839797 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1435'. It is not used by any node and should be removed from the model.
2023-06-15 08:53:03.673844928 [W:onnxruntime:, graph.cc:3211 CleanUnusedInitializers] Removing initializer '1376'. It is not used by any node and should be removed from the model.
/opt/rh/devtoolset-8/root/usr/include/c++/8/bits/stl_vector.h:932: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::operator[](std::vector<_Tp, _Alloc>::size_type) [with _Tp = int; _Alloc = std::allocator<int>; std::vector<_Tp, _Alloc>::reference = int&; std::vector<_Tp, _Alloc>::size_type = long unsigned int]: Assertion '__builtin_expect(__n < this->size(), true)' failed.
06/15 08:53:03 - mmengine - ERROR - tools/deploy.py - create_process - 82 - visualize onnxruntime model failed.
```

**Comment by jamestkpoon** [comment]
In addition, the .onnx seems to have an input size of [320,320]:
```
In [4]: session.get_inputs()[0].shape
Out[4]: [1, 3, 320, 320]
```

is it possible to make it follow the [300,300] I specify in training and deploy configs?

**Comment by irexyc** [comment]
Hi, @jamestkpoon 

**For the assert error**, there are some issues report that this problem occurs when using the sdk but we haven't find the cause. Your condition is different from them as you just use `onnxruntime` + `custom ops`. Can you try just remove the `/root/workspace/mmdeploy/mmdeploy/lib/libmmdeploy_onnxruntime_ops.so` file befoer converting the model?


**For the additional question**, tt's wired you onnx model has a input shape of 320 x 320. According to your model config, the input shape should be ['batch', 3, 'height', 'width'].

Normally, you don't have to change post_processing(nms) in codebase_config. The `model_inputs` in your backend_config is used by tensort backend and will take no effect for onnxruntime. Onnxruntime uses dynamic_axes to control whether the model support dynamic input shapes. When we convert the model, we need to supply a dummy input tensort to the pytorch model, If the `input_shape` is None, we will create the input according to `test_pipeline`, otherwise we will replace the `Reisze` transform with fixed resize. Without dynamic_axes, the model input shape will be same with dummy input shape when you convert the model. 

In short, you can just use the `configs/mmdet/detection/detection_onnxruntime_dynamic.py`.

**Comment by jamestkpoon** [comment]
@irexyc I failed to mention, that I tried ```configs/mmdet/detection/detection_onnxruntime_dynamic.py``` first before trying to make my own config, and unfortunately was getting the same error.

I am currently re-training the model using images that are 320x320, to see if the input size being a multiple of 32 improves things. This was inspired by a requirement of the [Megvii implementation](https://github.com/Megvii-BaseDetection/YOLOX).
After re-training, I will first try the provided deploy config again. 

Thanks for your reply!

**Comment by jamestkpoon** [comment]
@irexyc Unfortunately, same error with 320x320 image size. I've uploaded the model config, checkpoint, and a test image to https://drive.google.com/file/d/1hNGXogKI8ShNswtfXVZH9wYTQNDtcdmn/view?usp=sharing , could you please take a look? Thanks again.

**Comment by irexyc** [comment]
@jamestkpoon 

Sorry for late reply, have you removed /root/workspace/mmdeploy/mmdeploy/lib/libmmdeploy_onnxruntime_ops.so before converting the model?

**Comment by irexyc** [comment]
@jamestkpoon

What  onnxruntime are you using? We build the onnxruntime custom operator library under onnxruntime-1.8.1, may be you should install this specify version

**Comment by irexyc** [comment]
https://pypi.org/project/onnxruntime/1.8.1/#files

The onnxruntime 1.8.1 only support python in [3.6, 3.9]

**Comment by jamestkpoon** [comment]
@irexyc  1.8.1:

```
root@3c95d38328dc:~/workspace# python3
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime as ort
>>> ort.__version__
'1.8.1'
```

**Comment by irexyc** [comment]
If you can't run onnxruntime model with pure onnxruntime api, it might be issues with onnxruntime.

```python
import onnxruntime as ort
import numpy as np
# session_options = ort.SessionOptions()
# session_options.register_custom_ops_library('/path/to/libmmdeploy_ort_net.so') # path to libmmdeploy_onnxruntime_ops.so
session_options = None
sess = ort.InferenceSession('/path/to/onnx', session_options, ['CPUExecutionProvider'])
input = np.random.randn(1, 3, 224, 224).astype(np.float32) # b, c, h, w
output = sess.run(['output'], input_feed={'input': input})
```
